app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsActionMultisendFragment.kt

```diff
@@ -68,7 +68,7 @@ class TransactionDetailsActionMultisendFragment : BaseViewBindingFragment<Fragme
                                 value.data,
                                 value.dataDecoded?.let { paramSerializer.serializeDecodedData(it) },
                                 value.to.asEthereumAddressString(),
-                                balanceFormatter.formatAmount(value.value, true)
```

Comment:
Why is this different on master and release? And is false the right value for incoming here?

Reply:
https://app.zenhub.com/workspaces/safe-multisig---mobile-5c1b98250e13551e8aae81eb/issues/gnosis/safe-android/1041 was fixed directly on release to produce new rc